test/services/storage/storage.spec.ts

```diff
@@ -0,0 +1,85 @@
+import chai from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import sinonChai from 'sinon-chai'
+import dirtyChai from 'dirty-chai'
+import sinon from 'sinon'
+import { EventData } from 'web3-eth-contract'
+import Eth from 'web3-eth'
+import { SubstituteOf } from '@fluffy-spoon/substitute'
+import { Sequelize } from 'sequelize-typescript'
+
+import eventProcessor from '../../../src/services/storage/storage.processor'
+import { OfferService, StorageServices } from '../../../src/services/storage'
+import { sequelizeFactory } from '../../../src/sequelize'
+import Offer from '../../../src/services/storage/models/offer.model'
+
+chai.use(sinonChai)
+chai.use(chaiAsPromised)
+chai.use(dirtyChai)
+const expect = chai.expect
+
+const mockEventsFactory = (event: Record<string, unknown> = {}): EventData => Object.assign({
```

Comment:
There is pretty much same function here: https://github.com/rsksmart/rif-marketplace-cache/blob/master/test/utils.ts#L45

So please use this one so we don't replicate this unnecessarily.

Reply:
Sure, miss this one=)